### DISCUSSION TITLE
custom .cypress-cucumber-preprocessorrc.json file

### DISCUSSION THREAD

**Discussion by eramahatra** [badeball/cypress-cucumber-preprocessor#1260] [Q&A]
Hi guys
I'm struggling since 2 days to get cucumber working with another filename for the preprocessor config file which by default is _.cypress-cucumber-preprocessorrc.json_

Tried to load the custom file in my cypress config file :
 ` await preprocessor.addCucumberPreprocessorPlugin(on, config, {
    configFile: "custom-preprocessor.json",
  });
`
not working, it always look for the default file

tried several things but none working.

any idea ?

**Comment by badeball** [comment]
No such option like the one you're depicting exists. You can take a look at [cosmiconfig's](https://github.com/cosmiconfig/cosmiconfig) for documentation alternative places to place your config file.
